v2/datastream-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/datastream/ChangeEventConvertor.java

```diff
@@ -281,6 +281,12 @@ private static void populateMutationBuilderWithEvent(
           columnValue =
               Value.int64(ChangeEventTypeConvertor.toLong(changeEvent, colName, requiredField));
           break;
+        case FLOAT32:
+        case PG_FLOAT4:
+          // Replace `Value.float64` with `Value.float32` once client lib support float32.
```

Comment:
Please upgrade this dependency in the pom

Reply:
Done.